### DISCUSSION TITLE
Voice in Frontline

### DISCUSSION THREAD

**Discussion by travisterwilligar** [twilio-labs/paste#1350] [Office hours] [answered]
I'm looking to get general design feedback for voice calling in the Frontline app. Note that I'm focusing only on iOS for this round of validation, but Android will come later. 

Currently, Frontline is a mobile native app on iOS and Android and is currently in closed beta with only a few customers using it. The Frontline team has a number of customers who are either on the beta waitlist who have asked for voice calling, or customers who are only interested in using Frontline when voice calling is available.

Voice PM, Eng, and UX are guest engineering, PMing, and designing this feature on behalf of the frontline team.

Frontline has been designed using Paste tokens, but isn't currently using any paste components due to no current mobile native support. That being said, we'd like to stay in sync with paste, at least at a philosophical level wherever possible,

**The feedback I am looking for:**

- Overall usability of the calling flow from starting with either a contact or a conversation.
- Any parallel paste patterns or guidance we could potentially implement here.
- Iconography choices. Some of these icons don't exist in paste currently (there is one inconsistency in the design currently between voice mail controls and call controls on the dialer screen)
- Logging of call activity between after a call event has occurred in the flow of a conversation
- Receiving incoming calls
- Any other thoughts / feedback y'all have.

[Figma Prototype](https://www.figma.com/file/3QrLhKuNBU1EdgRjHNh09M/expl-Transfer-and-Voice?node-id=203%3A23)

**Comment by corinne-nagel** [comment, marked as answer]
Hey @travisterwilligar , thanks for sharing! The team will review before office hours this Thursday so we're ready.

**Reply by serifluous** [reply]
Here's the [video recording link](https://www.loom.com/share/94d5c5ab48e1421090dee77f72f282a8).

Let us know what icons y'all end up using, so we can add them to Paste. And please let us know what the icon asset requirements are for iOS & Android!
